### PR TITLE
added vertical format for text

### PR DESCRIPTION
**Pull request by BluewyDiamond**
There might be people who uses waybar in a vertical setup, thus having a vertical format would be very appreciated.

### CODE REVIEW

File: main.rs [hunk 1]
```diff
@@ -86,6 +86,11 @@ fn main() {
         None => "",
     };
 
+    let vertical_view = match args.iter().position(|arg| arg == "--vertical-view") {
+        Some(_) => true,
+        None => false
+    };
+
```
Comment (bjesus) [inline main.rs:93]: can we read this argument like we're reading the others? e.g.
```
    let ampm = args.iter().any(|arg| arg == "--ampm");
```

File: main.rs [hunk 2]
```diff
@@ -137,7 +142,11 @@ fn main() {
         .find(|(code, _)| *code == weather_code.parse::<i32>().unwrap())
         .map(|(_, symbol)| symbol)
         .unwrap();
-    let text = format!("{} {}", weather_icon, indicator);
+    let text = if vertical_view == true {
```
Comment (bjesus) [inline main.rs:145]: do we need this `== true` ?

**Review by bjesus — CHANGES_REQUESTED** [review]

**Comment by BluewyDiamond** [comment]
I have changed it according to the requests. Is it good?

**Comment by bjesus** [comment]
lgtm! thank you

**Comment by bjesus** [comment]
this feature will be deprecated and removed soon. please use    `wttrbar --custom-indicator '{ICON}\n{temp_C}'` instead.

**Pull request merged**